scripts/start.sh

```
@@ -13,45 +13,50 @@ if ! command -v docker-compose &> /dev/null; then
-    ORIGINAL_REDIS_HOST=$REDIS_HOST
-    
-    export DB_HOST=host.docker.internal
-    export REDIS_HOST=host.docker.internal
-    
-    export $(grep -v '^#' .env | grep -v '^DB_HOST=' | grep -v '^REDIS_HOST=' | xargs)
-    
-    echo "📝 Configuration Docker:"
-    echo "   - DB_HOST: localhost → host.docker.internal"
-    echo "   - REDIS_HOST: localhost → host.docker.internal"
+    # Exporter toutes les variables
+    export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)